src/main/resources/application.yml

```
@@ -28,3 +28,26 @@ spring:
+  metrics:
+    tags:
+      application: ${spring.application.name}
+      environment: ${spring.profiles.active}
+    enable:
+      jvm: true
+      process: true
+      http: true
+      system: true
+    distribution:
+      percentiles-histogram:
+        http.server.requests: true  # 히스토그램 활성화
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)